pallets/collator-allowlist/src/weights.rs

```diff
@@ -1,21 +1,21 @@
 //! Autogenerated weights for pallet_collator_allowlist
 //!
-//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 3.0.0
-//! DATE: 2021-06-18, STEPS: `[50, ]`, REPEAT: 100, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2021-12-05, STEPS: `100`, REPEAT: 200, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("altair-dev"), DB CACHE: 128
 
 // Executed Command:
 // target/release/centrifuge-chain
 // benchmark
 // --chain=altair-dev
-// --steps=50
```

Comment:
I would suggest keeping it to 50 and 100 since the higher the steps are the more the weight saturation leading to smaller weights

Reply:
Note that I didn't set these numbers, this is auto-generated. Why did the command running on my machine bump those numbers or how do you keep them at bay?

Reply:
> I would suggest keeping it to 50 and 100 since the higher the steps are the more the weight saturation leading to smaller weights

What does weight saturation mean here? Isn't this to have more information about possible outliers in the measurements?

Reply:
Anyway, those numbers are set here, so it seems off-scope to change them for this pallet's benchmarking:
https://github.com/centrifuge/centrifuge-chain/blob/rt4/scripts/run_benchmark.sh#L16-L17

Reply:
> What does weight saturation mean here? Isn't this to have more information about possible outliers in the measurements?
This means, when you run benchmarks over multiple times, your host does all sorts of caches that would make this runt faster. In the real world, we can predict the which code path the wasm will take since we cant predict the kind of transactions tht would come in. 
The more number of time you run the same code in benchmarks, the efficient the host if caching it and giving much smaller weights than it would ideally take on real world

Reply:
Thanks! Yes, makes sense.